src/test/regress/sql/gp_explain.sql

```diff
@@ -329,3 +335,36 @@ CREATE TABLE t1 (a int);
 EXPLAIN ANALYZE INSERT INTO t1 SELECT 1/gp_segment_id
 FROM gp_dist_random('gp_id');
 DROP TABLE t1;
+
+
+-- Check optional table alias support. When optimizer_enable_table_alias is off,
+-- table alias is not supported (this is default)
+-- start_ignore
+drop table if exists foo_alias;
+-- end_ignore
+
+-- check that with optimizer_enable_table_alias=off plan doesn't have aliases
+set optimizer_enable_table_alias=off;
+
+create table foo_alias (a int, b int);
+insert into foo_alias select generate_series(1,10);
+
+explain delete from foo_alias bbb using foo_alias aaa  where aaa.a=bbb.a;
+
+-- When optimizer_enable_table_alias is on, table alias is supported and it 
+-- can be easily seen with self joins
+set optimizer_enable_table_alias=on;
+explain delete from foo_alias bbb using foo_alias aaa  where aaa.a=bbb.a;
+
+-- Check that debug print of physical plan also contains alias
+set optimizer_print_plan=on;
+set client_min_messages=log;
+explain delete from foo_alias bbb using foo_alias aaa  where aaa.a=bbb.a;
+
+-- start_ignore
+drop table foo_alias;
+set optimizer_print_plan to default;
+set client_min_messages to default;
+-- end_ignore
+
+set optimizer_enable_table_alias to default;
```

Comment:
What if the order of tests will change, and tests that have been generated without aliases will fail?